src/parser/mod.rs

```diff
@@ -1044,6 +1046,16 @@ impl<'a> Parser<'a> {
         Ok(Statement::NOTIFY { channel, payload })
     }
 
+    pub fn parse_rename(&mut self) -> Result<Statement, ParserError> {
+        if self.dialect.supports_rename_table() && self.peek_keyword(Keyword::TABLE) {
+            self.expect_keyword(Keyword::TABLE)?;
+            let operations = self.parse_comma_separated(Parser::parse_rename_object_def)?;
+            Ok(Statement::RenameTable { operations })
+        } else {
+            self.expected("an object type after RENAME", self.peek_token())
```

Comment:
does it make sense to say e.g. 'table name' instead of 'object type'? Thinking the object type/def terminology in the sounds a bit generic if we're only dealing with tables in the context of this statement

Reply:
Good idea! Have Done.